woocommerce/woocommerce-ios
# Add icons to rows on the Card Reader settings screen

Closes #5289 

Add icons to the in-Person Payments Menu. I took the icons in Figma from the Android designs, for two reasons:
1. There is no "reader manual" icon for iOS
2. The iOS icons do not seem to be actual assets that can be extracted.

| Before | After - Light Appearance | After - Dark Appearance |
| ---- | ---- | --- |
| <img src="https://user-images.githubusercontent.com/2722505/141540510-ed5dee11-c68f-4308-853a-d166227d590b.png" width="250"/> | <img src="https://user-images.githubusercontent.com/2722505/141540455-84476305-f056-4533-a3fb-a7fb6eb19beb.png" width="250"/> | <img src="https://user-images.githubusercontent.com/2722505/141540474-42a0f608-106b-48da-b2e7-e059ef931a78.png" width="250"/> |

## Changes
* Added three new assets, declared images in the UIImage extension and added tests
* Touched the InPersonPaymentsMenuViewController to use a cell that seems more appropriate and to extract the localized strings to a private enumeration, following our own standards

## How to test
* Checkout the branch, build and run
* On a store eligible for In-Person Payments, navigate to Settings > In-person Payments
* Notice the icons

Update release notes:
